### ISSUE TITLE
Fitness function

### ISSUE THREAD

**Issue by sayakpaul** [rsteca/sklearn-deap#30]
Could you please shed some light on Fitness function that is being used here?

**Comment by rsteca** [comment]
Actually, you can choose which fitness function you would like to use in the "scoring" parameter. By default it uses the accuracy, so the individuals (or hyperparameters) that survive will be the ones with the best accuracy.
